console.log('errors');

try {
    identifier // Reference error
} catch {
    console.log('catched');
}
